api/reference/1.0/secrets/relationships/environment/list_related.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Fetch related
```

Comment:
We also need to create a page for the Environments relationship to Secrets to show how an Environment can retrieve its related secrets

Reply:
Done